packages/sdk-v2/src/sdk/analytics/mixpanel/api.native.ts

```diff
@@ -0,0 +1,19 @@
+import { Mixpanel } from "mixpanel-react-native";
```

Comment:
https://github.com/GoodDollar/GoodDAPP/pull/3886/files/2de46965eb287f858b3d1d07baccaf81a8dbae7f#r1097096172

Reply:
is this a bug?

Reply:
Yes, mutating prototype could cause bugs. Imagine if some other library also mutates prototype of mixpanel. You added own overrides but you see they aren’t working ? Why ? A lot of debugging and you found some library re-overriding the methods you need 
A lot of time spent. 
this is one of the reasons why developers agreed do not mutate prototypes as it could cause to unexpected consequences. You could override method at object instances. You could write wrapper class. There are many other opts to reach the same. Not always the easiest way is the most correct

Reply:
could cause is not "a bug"

Reply:
I understand but better try to avoid bugs in future, right ? You spent +5-10 min now but you possibly  saved hours in debugging in the future. As team lead I’m responsible for code deliveries from the team so if some potential bug will be missed and not fixed by myself and will turn onto a real bug, it will be part of my fault here too

Reply:
noted.

Reply:
I suppose you may have other point of view, but I used to do this way. At least to do things with quality and with some “margin of safety”. I do this automatically- saw it fixed it. 
I do not do things “looking fine” and 100% according some design/pattern anymore. But potential issues are still important for me